docs/beta.md

```diff
@@ -129,25 +128,6 @@ find it with `sudo find / -type f -name 'tools.jar'`.
 Note: you may wish to specify alternative JVM heap parameters `-Xmx`, `-Xms`, the
 values used in the example correspond to the JMXFetch defaults.
 
-## Systems
```

Comment:
preparing to kill this page, moving away contents that'll be useful after the GA release too